selfdrive/car/subaru/carcontroller.py

```diff
@@ -15,7 +15,11 @@ def __init__(self, car_fingerprint):
       self.STEER_DRIVER_ALLOWANCE = 60   # allowed driver torque before start limiting
       self.STEER_DRIVER_MULTIPLIER = 10   # weight driver torque heavily
       self.STEER_DRIVER_FACTOR = 1     # from dbc
-
+    if car_fingerprint in (CAR.OUTBACK, CAR.LEGACY):
+      self.STEER_DRIVER_ALLOWANCE = 300   # allowed driver torque before start limiting
+      self.STEER_DRIVER_MULTIPLIER = 1   # weight driver torque heavily
+      self.STEER_DRIVER_FACTOR = 1     # from dbc
+      self.STEER_DELTA_DOWN = 60         # torque decrease per refresh
```

Comment:
can't you use 70 like for the impreza? otherwise this will be hard to manage in panda safety

Reply:
The message length is 1 bit longer on pre-global. I don't currently know how to make it work, I think I cheated with the bit shifting for the legacy and outback to trim the lsb to match impreza length to not enable a violation. It would have been great if subaru kept the same CAN bus IDs but it's almost a separate make port